src/workflow/StreamlitUI.py

```diff
@@ -36,8 +36,7 @@ def upload_widget(
         key: str,
         file_types: Union[str, List[str]],
         name: str = "",
-        fallback: Union[List, str] = None,
-        symlink: bool = False,
```

Comment:
I decided to remove the symlink parameter for the method itself in favour of allowing the user to control it via a streamlit checkbox.